src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -124,6 +126,12 @@
   )
   private volatile Integer pyramidResolutions;
 
+  @Option(
+    names = {"-s", "--series"},
+    description = "Series index, if a single series should be converted"
+  )
+  private volatile int singleSeries = -1;
```

Comment:
Might be an idea to just go with `series` here of type `Integer` to allow for nullability and ease of detection if set.